e-tayfw/WorkFromHome
# feat: added new implementation for duplicate requests on same day

* feat(ci): add GitHub Actions workflow for Laravel CI/CD with deployment to EC2

* test: validate on push trigger backend CI/CD

* chore(env): add .env.testing and update CI workflow for testing environment

* chore(ci): dynamically generate .env and .env.testing

* test: validate on push trigger backend CI/CD

* chore(ci): update Laravel CI/CD workflow

* test: validate on push trigger backend CI/CD

* As an employee, I want to see my name so that I know I’m in the correct account (#29)

* feat + fix: Add name visible in Nav and Fix Auth routing issues

* feat: Front end fixes, shows users on that day clearer

* test: added automated frontend test case for Nav

* Fix: Fix routing issue and Fix styling of table in modal

* Fix: Styling issue with id number

---------



* chore(ci): update ci/cd workflow for backend (#31)

* chore(ci): update Laravel CI/CD workflow

* test: validate on push trigger backend CI/CD

* Update backend.yaml

* Create test.txt

* Update backend.yaml

* Update test.txt

* chore(ci): update ci/cd workflow for backend

---------
